app/src/main/res/layout/layout_contribution.xml

```diff
@@ -74,30 +74,31 @@
 
         <LinearLayout
             android:id="@+id/failed_image_options"
-            android:layout_width="0dp"
+            android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:orientation="horizontal"
-            android:layout_weight="1"
             android:visibility="gone"
             android:padding="@dimen/small_gap"
             >
-
             <ImageButton
-                android:id="@+id/cancelButton"
+                android:id="@+id/retryButton"
                 android:layout_width="wrap_content"
```

Comment:
this PR just swaps cancel button with retry button. This is not what is needed on issue.